python/w2dyn_cthyb/solver.py

```diff
@@ -687,16 +708,11 @@ def bs_diagflat(bs_array):
             self.G_tau, self.G_tau_error = w2dyn_ndarray_to_triqs_BlockGF_tau_beta_ntau(
                 gtau, self.beta, self.gf_struct)
 
-            self.G_iw = BlockGf(mesh=self.iw_mesh, gf_struct=self.gf_struct)
-
-            ### I will use the FFT from triqs here...
-            for name, g in self.G_tau:
-                bl_size = g.target_shape[0]
-                known_moments = np.zeros((4, bl_size, bl_size), dtype=complex)
-                for i in range(bl_size):
-                    known_moments[1,i,i] = 1
+            self.G_iw, self.G_iw_error = w2dyn_ndarray_to_triqs_BlockGF_iw_beta_niw(
+                giw, self.n_iw, self.beta, self.gf_struct)
 
-                self.G_iw[name].set_from_fourier(g, known_moments)
```

Comment:
@ahausoel Was there any particular reason why you used a Fourier transform instead of using the impurity Green's function from w2dynamics? Do the Legendre tails from w2dynamics pose a problem anywhere?

@HugoStrand What is your opinion on that?

Reply:
If the Legendre tail are undesirable we could set `config["General"]["FTType"] = "plain"`.